### DISCUSSION TITLE
Current versions of some remark plugins do not work with mdsvex

### DISCUSSION THREAD

**Discussion by jokull** [pngwn/MDsveX#236] [General]
I tried installing `remark-footnotes` but 3.0 (the most recent version) doesn't work. Dropping down to 2.0 did the trick. Let us know if you run into any other similar issues/solutions.

**Comment by pngwn** [comment]
This is because of some significant changes in a recent version of the libraries `mdsvex` is built on top of. Older version of these plugins work with `mdsvex` as they always have done.

If a plugin you want to use isn't working, check the readme of that plugin and it should mention whether or not it is compatible with the recent changes. For example [remark-footnotes](https://github.com/remarkjs/remark-footnotes#important) has this message:

> This plugin is affected by the new parser in remark (micromark, see remarkjs/remark#536). Use version 2 while you’re still on remark 12. Use version 3 for remark 13+.

**Comment by matzhaugen** [comment]
Is there any way to bump the underlying versions mdsvex is built on, specifically to get access to `remark-gfm`, which seems to have this problem as well.

**Reply by webel** [reply]
did you find a solution? I've tried older versions of remark-gfm without success

**Reply by ZachSaucier** [reply]
I was only able to get it working with `remark-footnotes` version 2: `npm i --save-dev remark-footnotes@2.0`

**Reply by einarpersson** [reply]
Does remark-gfm work with MDsveX now, in 2025?

**Reply by lea-8** [reply]
Not for footnotes, don't know about other GFM features. For footnotes [this](https://github.com/pngwn/MDsveX/discussions/223) solved it for me, although I do agree `remark-gfm` footnotes should be supported especially since its marked with 🟢 on the plugin page.
